epregressions/runtests.py

```diff
@@ -200,7 +200,13 @@ def add_or_modify_output_sqlite(idf_text, force_output_sql: ForceOutputSQL,
             raise ValueError("Expected an Enum ForceOutputSQLUnitConversion, not "
                              "{}".format(force_output_sql_unitconv))
 
-        if isEpJSON:
+        # special ugly case for handling unit testing -- note that the unit testing here is based around a "dummy"
+        # energyplus which reads in a small JSON configuration blob, even though it thinks it is an IDF.  This confuses
+        # this function, so I'll put in a small trick to just let the code pass through
+        if idf_text.startswith('{"config"'):
+            return idf_text
```

Comment:
Funny little pass-through specifically for unit testing.  Not my favorite, but it's OK.